### DISCUSSION TITLE
How to make VCPKG copies dependencies pdb files in output directory

### DISCUSSION THREAD

**Discussion by Belox86** [microsoft/vcpkg#37828] [Q&A]
Hi everyone,
i have multiple "internal" library which i'm trying to manage with vcpkg+cmake. 
Generally speaking everything is working properly. 
I have only one minor issue which i can't solve. When working in a project that uses my internal dependecies (in Windows with MSVC compiler) i can't have the pdbs of my internal dependencies copied along with the dlls.

The dependencies have the vcpkg_copy_pdbs() in the portfile and they properly export the pdbs in the local cache ..\build\vcpkg_installed\x64-windows\debug\bin along with the dlls. it just miss the final step to have them copied where the executable files is placed so that dependencies symbol can be load. 
Note that the dependecies dlls are copied from the local cache to the output directories and the program starts properly. Also the pdb of the project itself are in the output directory. Only the pdb of the dependecies are missing. 
What is the proper way to handle this? It seems strange to me that vcpkg handle everything and then it just miss the final copy. 
Thank you very much. 
Michele

**Comment by derekpiasecki** [comment]
> The dependencies have the vcpkg_copy_pdbs() in the portfile and they properly export the pdbs in the local cache ..\build\vcpkg_installed\x64-windows\debug\bin along with the dlls. it just miss the final step to have them copied where the executable files is placed so that dependencies symbol can be load. Note that the dependecies dlls are copied from the local cache to the output directories and the program starts properly. Also the pdb of the project itself are in the output directory. Only the pdb of the dependecies are missing. 

It looks like this same question was asked here, and there wasn't an official solution, but you can find the [installed folder](https://learn.microsoft.com/en-us/vcpkg/reference/installation-tree-layout) from which to copy them, ~via a workaround mentioned in https://github.com/microsoft/vcpkg/issues/8106.~ It seems that workaround to find the path is not needed, at least not in manifest mode, as the PDBs are in the `$(VcpkgBin)` folder.
